xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Code/Translations.xml

```diff
@@ -192,6 +192,7 @@ help.videos.videoCard5.topic2=Learn how to use your application
 help.videos.videoCard6.title=Add / Remove Extensions
 help.videos.videoCard6.topic1=Discover how to add/remove extensions using the Extension Manager
 help.videos.videoCard6.topic2=Discover the XWiki Administration UI
+help.videos.hint=The following documentation links provide an alternative to this video.
```

Comment:
Maybe we should make this an actual description of the video? Like saying that you see a screencast showcasing the features that are listed below without audio (which could be interesting information for a blind user)?

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2466/commits/758d90df8de85d58a87bea96d3438f98e33f458d :+1: